src/main/java/net/openhft/chronicle/network/api/TcpHandler.java

```diff
@@ -52,6 +52,7 @@ default void onWriteTime(long writeTimeNS,
                              final int position) {
     }
 
+    @Deprecated(/* to be removed in x.24 */)
```

Comment:
Sonar is telling me you should use the @deprecated annotation in javadoc, and I tend to agree, I'm pretty sure whatever text you put in that will appear when you roll over the method in IntelliJ. I think that "to be removed" perhaps with a reason should go there.

Reply:
Basically, it could replace the comment within the `@Deprecated(...)` method annotation.

Reply:
Agree but could not get consensus from the team :)